user/tests/wiring/no_fixture/thread.cpp

```diff
@@ -262,8 +262,8 @@ test(APPLICATION_WATCHDOG_01_fires_timeout)
 {
 	s_ram_free_before = System.freeMemory();
 	timeout_called = 0;
-	ApplicationWatchdog wd(5, timeout);
-	HAL_Delay_Milliseconds(10);
+	ApplicationWatchdog wd(50, timeout);
+	HAL_Delay_Milliseconds(60);
```

Comment:
@avtolstoy Is it reasonable to increase the timeout to 100msec? This particular test on my trackerM passes more consistently with increased delay.

Reply:
No, this value needs to be under 2 * timeout value close to the actual timeout, this is the exact issue we are trying to fix.